### DISCUSSION TITLE
I have installed the extension, but the three dots does not appear for Launching CodeViz

### DISCUSSION THREAD

**Discussion by WorrenGilgamesh** [vizzuhq/codeviz#19] [Q&A] [answered]
I have restarted my visual code, and nothing happens. The VS Code Counter extension is working.

**Comment by simzer** [comment]
Hi, we moved the codeviz launch menu to the exprorer's context menu because there were multiple request to remove it from the toolbars.

**Reply by WorrenGilgamesh** [reply]
How should I access it?

**Reply by simzer** [reply, marked as answer]
There are two ways:
- open the command palette with ctrl-shift-P and type codeviz
- or from the popup menu of the folders in explorer (this will open a filtered view showing only the selected folder)

**Reply by WorrenGilgamesh** [reply]
Thanks! For some reason, the first time I tried codeviz command didn't work
